compiler/il/OMRNode.hpp

```diff
@@ -1812,7 +1808,6 @@ class OMR_EXTENSIBLE Node
       visitedForHints                       = 0x00000080, ///< Used only during codegen phase
       nodeIsNonNegative                     = 0x00000100,
       nodeIsNonPositive                     = 0x00000200,
-      nodeIsDebug                           = 0x00000400, ///< Should not be counted for metrics
```

Comment:
Do you want to announce this flag's new availability by not deleting it outright but adding an availability comment instead?  e.g.,

```
// Available        =0x00000400,
```

Reply:
I do not think this is a good idea. We have the starts of a discussion starting in https://github.com/eclipse/omr/issues/2297 which deal with problems of overloading flag values and getting unexpected behavior. One has to think and investigate very hard before adding a new flag overload at the moment. We should be looking for alternative solutions to this broad problem and encouraging more overloading of flag values is in my opinion dangerous.

I would prefer for us to start having less overloading moving forward until we setting on a more general solution to this not so trivial problem at hand. Do you agree with this direction?

Reply:
I absolutely fully support an overall effort to reduce node flag overloading.  I've been been burned by this confusion in years past.   In this case, I was fooled by the comment preceding this enum which puts `0x00000400` in the "general node flags" category which I incorrectly assumed it meant it couldn't/shouldn't be overloaded, but that's clearly not true.  Your code is fine as is.